src/createServer.js

```diff
@@ -1,3 +1,74 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+
+    const textToConvert = normalizeUrl.pathname.slice(1);
+
+    const params = Object.fromEntries(normalizeUrl.searchParams.entries());
+
+    const toCase = params.toCase;
+
+    const cases = [
+      'SNAKE',
+      'KEBAB',
+      'CAMEL',
+      'PASCAL',
+      'UPPER',
+    ];
+
+    const ArrOfErrors = [];
+
+    res.setHeader('Context-Type', 'application/json');
+
+    if (!textToConvert) {
+      ArrOfErrors.push({
+        message: 'Text to convert is required. Correct request is: '
+        + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    };
+
+    if (!toCase) {
+      ArrOfErrors.push({
+        message: '"toCase" query param is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (!cases.includes(toCase)) {
+      ArrOfErrors.push({
+        message: 'This case is not supported. '
+        + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+      });
+    }
```

Comment:
Maybe it's better to move URL validation to separate function.

Reply:
Completely agree, move URL validation logic to the separate function